basic/src/test/java/org/jboss/hal/testsuite/test/configuration/singleton/SingletonPolicyTestCase.java

```diff
@@ -198,11 +207,17 @@ public void editQuorum() throws Exception {
                 .clickLabel();
 
         final int value = 5;
-        new ConfigChecker.Builder(client, SINGLETON_POLICY_ADDRESS)
-                .configFragment(page.getConfigFragment())
-                .editAndSave(ConfigChecker.InputType.TEXT, QUORUM, String.valueOf(value))
-                .verifyFormSaved()
-                .verifyAttribute(QUORUM, value);
+
+        try {
+            new ConfigChecker.Builder(client, SINGLETON_POLICY_ADDRESS)
+                    .configFragment(page.getConfigFragment())
+                    .editAndSave(ConfigChecker.InputType.TEXT, QUORUM, String.valueOf(value))
+                    .verifyFormSaved()
+                    .verifyAttribute(QUORUM, value);
+        } catch (NoSuchElementException e) {
+            //TODO remove this when issue is resolved
```

Comment:
Please remove the TODO comment. We don't want to remove regression assertion messages.